backend/cmd/migrate/migrations/000013_create_feedback_table.up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE feedback (
```

Comment:
can we normalize the feedback to a rating out of 100 and store that in addition to the raw value?

Reply:
to double check this is what you had in mind -- create a rating enum (1-3 for now) & then have the request send a rating obj like `rating: {score: 3, label: "great"}`?